webpack/assets/javascripts/react_app/common/document.js

```diff
@@ -0,0 +1,8 @@
+import { runningInPhantomJS } from './helpers';
+
+/**
+ * Whether or not the page is focused (beeing used atm)
+ * @return {boolean}
+ */
+export const doesDocumentHasFocus = () =>
+  runningInPhantomJS() || (document.hasFocus ? document.hasFocus() : true);
```

Comment:
is it possible to mock `document.hasFocus` rather then referring to the js driver in a non test file?

Reply:
I try to find ways of doing that but it's is not that simple with capybara + phantomjs :-1:

Reply:
Or is the test without braces if the function exists?

Reply:
If `runningInPhantomJS` it will return `true`.
If the `document.hasFocus` method exists, it will return its result.
If the `document.hasFocus` does not exist, it will return `true`.

In other words, the only way to get `false` is if the `document.hasFocus()` will return `false`.

Reply:
Right, initially I missed the braces vs no-braces. Guess I did too much ruby development lately :)